### DISCUSSION TITLE
Use with Jakarta Faces

### DISCUSSION THREAD

**Discussion by harvec** [primefaces/community#356] [PrimeFaces Blocks]
Is there a version of PrimeFaces Blocks that can be used with Jakarta EE 9.1 and 10? If not, is there an ETA?

**Comment by melloware** [comment]
It should work fine if you upgrade the PrimeFaces blocks project using this guide. 

https://www.melloware.com/java-ee-6-8-to-jakarta-ee-10-upgrade/
